components/language-switcher.tsx

```
@@ -0,0 +1,173 @@
+                <div className="
+                  flex items-center justify-center 
+                  w-5 h-5 
+                  bg-blue-100 dark:bg-blue-800 
+                  rounded-full
+                ">
+                  <svg 
+                    className="w-3 h-3 text-blue-600 dark:text-blue-300" 
+                    fill="currentColor" 
+                    viewBox="0 0 20 20"
+                  >
+                    <path 
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)